src/lib.rs

```diff
@@ -41,6 +41,7 @@ mod graph;
 pub mod iterators;
 mod vertex_id;
 
+#[cfg(feature="graphviz")]
```

Comment:
I think the feature name "dot" would be better and be in line with the dot crate feature.

Reply:
I tried it out. But the dependency's name is also `dot`. Dependencies and features must not have the same name.

Reply:
Ok, we can leave it like this then.

Reply:
Just a note, since the dependency "dot" is optional, there is already a "feature" called "dot" to have the crate be included so the line `#cfg(feature="dot")] should still work, if I'm not mistaken.

Reply:
@Dynisious you're right. My mistake was that I must not define `dot` in `features` table when I have a optional dependency. 

@octavonce I renamed it. If the travis build passes, the PR is ready to be merged :)